Neos.SiteKickstarter/Resources/Private/Generator/Fusion/NodeTypes/Pages/Page.fusion

```diff
@@ -0,0 +1,11 @@
+prototype(Neos.Neos:Document.Document) < prototype({packageKey}:DefaultPage) {
```

Comment:
This looks somehow wrong, imho this should read:
`prototype({packageKey}:Page)) < prototype(Neos.Neos:Document) {`
If we do this we have to add the NodeType for that aswell.

Reply:
Also i would like to create to a `Site` and a `Page` NodeType by default, with constraints that allow only page and shortcut inside root.  That would require two fusion prototypes aswell.